apps/chrome-devtools/src/manifest.json

```diff
@@ -7,6 +7,7 @@
   "permissions": [
     "activeTab",
     "scripting",
+    "storage",
```

Comment:
Do not forget to add a justification to this permission to the Chrome Developer Portal when publishing this new version (Otherwise the publication maybe rejected)